db/v-2/tables/lnk-user-project.xml

```diff
@@ -31,4 +31,8 @@
     </rollback>
   </changeSet>
 
+  <changeSet id="lnk-user-project-2" author="frolov">
+    <renameTable newTableName="lnk_user_project" oldTableName="lnk-user-project" />
```

Comment:
Since you are anyway rename it, may be add absent `i`? Also rename corresponding classes? I don't see the real reason to shorten this word, the economy just in one char symbol, however it harder to read

```suggestion
    <renameTable newTableName="link_user_project" oldTableName="lnk-user-project" />
```